packages/central-server/src/tests/scaffolding.test.js

```diff
@@ -43,6 +40,6 @@ before(async () => {
 after(async () => {
   const models = getModels();
   SendEmail.sendEmail.restore();
-  await clearTestData(models.database, testStartTime);
+  await clearAllTestData(models.database);
```

Comment:
What happened here? Are there just too many different types of records being created for us to clean up the old way?

This `clearTestData()` function has usually worked surprisingly well, though it ought to be a tad brittle. Wondering about what issues it may have.

Reply:
Ah yep. BTW I started adding second lines to commit messages with notes. This one is "clearnAllTestData simply truncates all tables now that we use tupaia_test instead of the main tupaia db". 

However that doesn't tell the full story... The reason this was necessary here is because the importer creates a bunch of different records across a few different models. The tests would need some heavy mocking to make sure every model creates records with test ids. I think in this case specifically something like survey_screen was being created without a test id.

The alternative of just truncating all tables seemed much simpler, and is equivalent in terms of behaviour, thanks to your recent migration to using a separate db!